usr/src/test/zfs-tests/tests/functional/inheritance/inherit_001_pos.ksh

```diff
@@ -331,7 +331,7 @@ function scan_state { #state-file
 				if [[ $op == "-" ]]; then
 					log_note "No operation specified"
 				else
-					export __ZFS_POOL_RESTRICT="$TESTPOOL"
+					export __ZFS_POOL_RESTRICT="TESTPOOL"
```

Comment:
I wasn't the original author, and haven't had time to double check this specific change. I think it was added due to failures seen in this specific test case, but I see that we already ignore failures of this test in our testing (see https://github.com/openzfs/openzfs-ci/blob/master/jenkins/sh/run-zfstest/zfstest-report.py#L51), so maybe this should be removed from the diff?

@jwk404 do you remember why this line was changed?